Source/WebCore/platform/graphics/PixelBufferConversion.cpp

```diff
@@ -316,9 +318,179 @@ static void copyImagePixels(const ConstPixelBufferConversionView& source, const
 }
 #endif
 
+#if ENABLE(PIXEL_FORMAT_RGBA16F)
+static Float16 readFloat16(const std::span<const uint8_t>& span8, size_t offset)
+{
+    union {
+        Float16 float16 { };
+        uint8_t bytes[sizeof(Float16)];
+    } magicBox;
```

Comment:
magicBox seems like an interesting name but it’s a local union so that seems fine